requirements.txt

```diff
@@ -1,17 +1,21 @@
 alembic==1.8.0
 cloudpickle==2.0.0
 dask[distributed]==2022.6.0
+
+fastapi==0.79.0
 flask==2.0.1
 flask-cors==3.0.10
 flask-socketio==5.1.1
 furl==2.1.3
 networkx==2.5
 psutil==5.9.0
+python-socketio
```

Comment:
needs to be pinned to a particular version

Reply:
@wjcunningham7  - All comments will be responded to and fixed as part of the v11 release. Some of these were in our internal list to fix as part of the next release.

Reply:
pinned to a particular version